docs/reference/migration/migrate_8_0/discovery.asciidoc

```diff
@@ -37,3 +39,7 @@ are no longer supported. In particular, this includes:
 - `discovery.zen.master_election.wait_for_joins_timeout`
 - `discovery.zen.master_election.ignore_non_master_pings`
 - `discovery.zen.publish.max_pending_cluster_states`
+
+*Impact* +
+To avoid errors, unset and discontinue use of any `discovery.zen.` settings.
```

Comment:
"unset and discontinue use" seems redundant. Again, I'd lead with the recommended action, something like:

> Discontinue using the `discovery.zen` settings. Specifying these settings in `elasticsearch.yml` will result in an error on startup.